hack/errata.py

```diff
@@ -90,6 +92,58 @@ def notify(message, webhook=None):
     }).encode('utf-8'))
 
 
+def get_open_prs_to_fast(repo):
+    pulls = repo.get_pulls(state='open', sort='created', base='master')
+    for pr in pulls:
+        try:
+            # Skip unknown PRs
+            if not pr.title.startswith("Enable "):
+                continue
+            # Ignore PRs which don't target fast
+            if pr.title.split(" ")[3] != "fast":
+                continue
+            yield pr.number, pr.body
+        except Exception as e:
+            print(f"Failed to parse {pr.number}: {e}")
+
+
+def extract_errata_number_from_body(body):
+    ERRATA_MARKER = 'https://errata.devel.redhat.com/advisory/'
+    first_line = body.split('\n')[0]
+    links = [
+        x for x in first_line.split(' ') if x.startswith(ERRATA_MARKER)
+    ]
+    if len(links) == 0:
+        return None
+    errata_num = links[0].split('/')[-1]
```

Comment:
nit: `rsplit('/', 1)`?

Reply:
Fixed in 212d2366324e63f49de66ca5c740fa9c162478c8